src/lib/annotations/doc/CreateHighlightDialog.js

```diff
@@ -323,6 +344,7 @@ class CreateHighlightDialog extends EventEmitter {
         highlightDialogEl.addEventListener('click', this.stopPropagation);
         highlightDialogEl.addEventListener('mouseup', this.stopPropagation);
         highlightDialogEl.addEventListener('touchend', this.stopPropagation);
+        highlightDialogEl.addEventListener('touchstart', this.stopPropagation);
```

Comment:
There's some logic in DocAnnotator that runs through and hides this, on touchstart